### ISSUE TITLE
How does the num_radar_pts parameter is obtained?

### ISSUE THREAD

**Issue by HYY-ying** [nutonomy/nuscenes-devkit#1098]
Hello! I would like to ask how the num_radar_pts parameter is obtained? Is it to convert the lidar bounding box and radar point cloud data into the same coordinate system and then use the function points_in_box from nuscenes.utils.geometry_utils to count? And how many radar sweeps were used? The result I got by converting the lidar bounding box and radar pcd data into the global coordinate and counting pts using function points_in_box from nuscenes.utils.geometry_utils is different from the num_radar_pts given in the sample_annotation. I only used just a single radar sweep and didn't do any invalid point filtering.

**Comment by whyekit-motional** [comment]
@HYY-ying there are multiple radars on the ego, so you might have to consolidate all the points for all the radars at a given keyframe and then use that consolidated radar point cloud for the counting

**Comment by HYY-ying** [comment]
Thank you for your answer! I have one more question, what is the relationship between radial velocity V_r and V_x(V_y)?

**Comment by HYY-ying** [comment]
And what's the definition of steering angle? Is it α or (π/2-α) in the image?
![QQ图片20240801154350](https://github.com/user-attachments/assets/eee37733-f6a4-4b85-ae55-1ca372eae367)

**Comment by HYY-ying** [comment]
yep, suppose this picture is in radar coordinate system, I'm trying to figure out if the steering angle is (π/2-α) in nuscenes? I think so, but I'm not sure.
![image](https://github.com/user-attachments/assets/aca28f21-f55c-4bec-86ed-ee573cc8112a)
